_ml-commons-plugin/pretrained-models.md

```diff
@@ -66,12 +66,12 @@ The following table provides a list of sparse encoding models and artifact links
 
 Cross-encoder models support query reranking. 
 
-The following table provides a list of cross-encoder models and artifact links you can use to download them. 
+The following table provides a list of cross-encoder models and artifact links you can use to download them. Note that you must prefix the model name with `huggingface/cross-encoders`, as shown in the **Model name** column. 
```

Comment:
> Note that you must prefix the model name with `huggingface/cross-encoders`, as shown in the **Model name** column.

idk if this sentence is necessary since it's documented in the table. As a user I copy/paste the model name out of the table and stick it in the pretrained model _register body, so as long as the thing in the table is correct, I'm happy.

Reply:
We have the same sentence for transformer models. Mainly because when you look at the model config, there is a `name` field that does not include the full prefix (for example, `cross-encoder/ms-marco-MiniLM-L-6-v2`). So users might be confused why the `name` in the table is not the same as the `name` in the model config.